src/constants/alertMessages/index.ts

```diff
@@ -0,0 +1,13 @@
+import {
+  CARD_NUMBER_MAX_LENGTH,
+  CARD_OWNER_NAME_MAX_LENGTH,
+  MONTH_RANGE,
+} from "../variables";
+
+export const INPUT_ONLY_NUMBER = "숫자만 입력해 주세요.";
+
+export const OVER_CARD_NUMBER_MAX_LENGTH = `카드 번호는 최대 ${CARD_NUMBER_MAX_LENGTH}자리만 입력할 수 있습니다.`;
+
+export const OVER_MAX_MONTH = `월은 최대 ${MONTH_RANGE.max}까지만 입력 가능 합니다.`;
+
+export const OVER_CARD_OWNER_NAME_MAX_LENGTH = `이름은 최대 ${CARD_OWNER_NAME_MAX_LENGTH}까지만 입력 가능 합니다.`;
```

Comment:
```suggestion
export const CARD_VALIDATION_ERROR_MESSAGES = {
  NUMBER_ONLY: "숫자만 입력해 주세요.",
  CARD_NUMBER_MAX_LENGTH: `카드 번호는 최대 ${CARD_NUMBER_MAX_LENGTH}자리만 입력할 수 있습니다.`,
  MAX_MONTH: `월은 최대 ${MONTH_RANGE.max}까지만 입력 가능 합니다.`,
  NAME_MAX_LENGTH: `이름은 최대 ${CARD_OWNER_NAME_MAX_LENGTH}까지만 입력 가능 합니다.`,
};
```

상수 네이밍만 보고 에러메세지라고 인지하기 힘들 것 같아서 위처럼 해보면 어떨까요?